### PR TITLE
moved to 5.34

### PR DESCRIPTION
**Pull request by katietz**

### CODE REVIEW

**Review by varlackc — COMMENTED** [review]
We probably should reset the `build number` since the package is not in defaults
https://github.com/AnacondaRecipes/perl-feedstock/blob/f96ce20e16ffb40f36a8c25ff490fd9d6e510047/recipe/meta.yaml#L11

File: recipe/build.sh [hunk 1]
```diff
@@ -1,102 +1,44 @@
...
+    _config_args+=("-Dsysroot=$(dirname $(dirname ${CC}))/$(${CC} -dumpmachine)/sysroot")
+  else
+    _config_args+=("-Dsysroot=${CONDA_BUILD_SYSROOT}/usr")
+  fi
+fi
 
-./Configure -de "${_config_args[@]}"
+./Configure "${_config_args[@]}" \
+                -de
 make
```
Comment (skupr-anaconda) [inline recipe/build.sh:42]: Can we use `make -j${CPU_COUNT}`?

**Review by skupr-anaconda — APPROVED** [review]
LGTM

File: recipe/build.sh (hunk 1 above)
Comment (katietz) [reply recipe/build.sh:42]: hmm, I try to avoid the -j option, as it might lead to too many threads running .... eg on ppc64le

Comment (katietz) [reply recipe/build.sh:42]: I use the build-number 1 by intention, as there is already for osx-arm64 the build-number 0 available.

Comment (katietz) [reply recipe/build.sh:42]: you could find out by either looking on zeus directly, or by checking via 'CONDA_SUBDIR=osx-arm64 conda search perl=5.34'

**Review by varlackc — APPROVED** [review]
LGTM

**Pull request merged**